pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -160,26 +154,25 @@ def instance_group():
 @add_options(include_classorigin_instance_option)
 @add_options(propertylist_option)
 @add_options(namespace_option)
-@add_options(interactive_option)
 @click.pass_obj
 def instance_get(context, instancename, **options):
     """
     Get an instance of a class.
 
     The instance can be specified in two ways:
 
-    * By specifying an untyped WBEM URI of an instance path in the INSTANCENAME
-      argument. The namespace in which the instance is looked up is the
-      namespace specified in the WBEM URI, or otherwise the namespace specified
-      in the --namespace option, or otherwise the default namespace of the
-      connection. Any host name in the WBEM URI will be ignored.
+    1. By specifying an untyped WBEM URI of an instance path in the INSTANCENAME
+    argument. The namespace in which the instance is looked up is the
+    namespace specified in the WBEM URI, or otherwise the namespace specified
+    in the --namespace option, or otherwise the default namespace of the
+    connection. Any host name in the WBEM URI will be ignored.
 
-    * By specifying the --interactive option and a class name in the
-      INSTANCENAME argument. The instances of the specified class are displayed
-      and the user is prompted for an index number to select an instance.
-      The namespace in which the instances are looked up is the namespace
-      specified in the --namespace option, or otherwise the default namespace
-      of the connection.
+    2. By specifying a class name with the characters ".?" as the keys in the
```

Comment:
On "By specifying a class name with the characters ".?" as the keys in the INSTANCENAME argument": I don't find this an intuitive description of the syntax. Also, since we introduced the term "wildcard keys" i the docs, it makes sense to also use it in the help text.

How about: "By specifying a class name with wildcard keys in the INSTANCENAME argument, i.e. "CLASSNAME.?""

Occurs multiple times.

Reply:
DONE